api/swagger.yaml

```diff
@@ -295,4 +306,27 @@ paths:
           description: "Not Found"
         "500":
           description: "Internal Server Error"
-
+definitions:
```

Comment:
The definition for `KubernetesProvider` should also contain the `permissions` key.

In JSON it looks like this:
```json
  "permissions": {
    "read": [
      "test-read-group"
    ],
    "write": [
      "test-write-group"
    ]
  }
  ```
In the code these are actually separate structs, so it's fine if you want to create them as separate definitions or define them as part of the same definition.

Reply:
Defined as a separate object for future reuse